lua/lvim/core/lualine/components.lua

```diff
@@ -123,9 +123,8 @@ return {
       local supported_linters = linters.list_registered(buf_ft)
       vim.list_extend(buf_client_names, supported_linters)
 
-      local unique_client_names = vim.fn.uniq(buf_client_names)
-
-      local language_servers = "[" .. table.concat(unique_client_names, ", ") .. "]"
+      local unique_client_names = table.concat(vim.fn.uniq(buf_client_names), ", ")
```

Comment:
while we're at it, `vim.fn.uniq` might be unnecessary, I don't think `get_active_clients` returns duplicates

Reply:
done